benchmarks/gbench/common/distributed_vector.cpp

```diff
@@ -163,19 +163,19 @@ void check_reduce(T actual) {
   }
 }
 
-template <class... BackendT> static void Reduce_DR(benchmark::State &state) {
-  T actual{};
-  xhp::distributed_vector<T, BackendT...> src(default_vector_size, fill);
-  Stats stats(state, sizeof(T) * src.size(), 0);
-  for (auto _ : state) {
-    for (std::size_t i = 0; i < default_repetitions; i++) {
-      stats.rep();
-      actual = xhp::reduce(src);
-    }
-  }
-  check_reduce(actual);
-}
-DR_BENCHMARK(Reduce_DR);
+// template <class... BackendT> static void Reduce_DR(benchmark::State &state) {
```

Comment:
do you need to comment out whole test? at least compilation may still be checked and we will have original git history